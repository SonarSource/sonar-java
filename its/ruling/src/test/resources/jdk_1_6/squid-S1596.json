{
'jdk_1_6:java/awt/KeyboardFocusManager.java':[
376,
390,
],
'jdk_1_6:java/awt/dnd/DragGestureEvent.java':[
303,
],
'jdk_1_6:java/net/URLConnection.java':[
561,
1096,
],
'jdk_1_6:java/security/Security.java':[
1043,
],
'jdk_1_6:java/security/cert/CollectionCertStoreParameters.java':[
81,
],
}
