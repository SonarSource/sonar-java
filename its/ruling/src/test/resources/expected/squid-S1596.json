{
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2StAXEventWriter.java':[
401,
402,
],
'project:java/awt/KeyboardFocusManager.java':[
433,
447,
],
'project:java/awt/dnd/DragGestureEvent.java':[
347,
],
'project:java/net/URLConnection.java':[
598,
1156,
],
'project:java/security/Security.java':[
1081,
],
'project:java/security/cert/CollectionCertStoreParameters.java':[
98,
],
'project:javax/security/auth/x500/X500Principal.java':[
121,
],
'project:javax/xml/crypto/dsig/spec/ExcC14NParameterSpec.java':[
74,
],
'project:javax/xml/crypto/dsig/spec/XPathFilterParameterSpec.java':[
69,
],
'project:javax/xml/crypto/dsig/spec/XPathType.java':[
130,
],
}
